docs/proposal-rose-suite-run.md

```diff
@@ -149,3 +196,38 @@ a single command (e.g. `cylc flow`?) to:
 * Warm start a suite.
 * Cold start a suite.
 * Clean start a suite.
+
+## Work Plan
+([PR] identifies a pull request to a development branch)
+- [ ] Identify which tests should work in a different way
+  - [ ] Tests currently in the Rose Codebase
+  - [ ] Tests currently in the Cylc Codebase
+- [ ] Create a development branch of master for this set of
+  changes.
+- [ ] [PR] Agree modifications to tests which will check
+  that these changes have worked.
+- [ ] Identify aspects changes not covered by tests
+  - [ ] Create tests or explain why tests cannot be created
+- [ ] [PR] Implement the new `suite.rc` & `flow.rc` parsing
```

Comment:
Doing the rename as part of this change is not necessary, but will make some sense.

We should call all of these files `flow.rc` (or `cylc-flow.rc`) and they will share the same schema. The suite `flow.rc` will use the full schema. The site/user `flow.rc` will use a reduced version of the same schema?

Reply:
Will we force the reduction of that schema? Or would the same schema be available to both if for some weird reason a site wanted to add a default suite graph (say) to their `global-flow.rc`. 

I had something like this in mind:
![flow_rc_settings_locs](https://user-images.githubusercontent.com/26465611/62034412-e179e500-b1e5-11e9-9abe-b8c321e77efe.png)

Reply:
He's not suggesting that, according to my reading.